src/util/restoreArchiveNote.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+export const restoreArchiveNote = async (_id, token, noteDispatch) => {
+  try {
+    const response = await axios({
+      method: 'post',
+      url: `/api/archives/restore/${_id}`,
+      headers: { authorization: token },
+    });
+    console.log(response);
```

Comment:
You could remove the unwanted `console.log()`'s

Reply:
Thank you Vishal for pointing out the unwanted ```console.log```.